kraken-build/src/kraken/std/python/pyproject.py

```diff
@@ -106,14 +106,14 @@ def to_toml_string(self) -> str:
         return tomlkit.dumps(self.data)
 
 
-class PyprojectHandler(ABC):
+class TomlConfigHandler(ABC):
```

Comment:
I don't think this rename makes sense. The class is very much tied to Pyprojects.

Reply:
Makes sense. Should we add a hard check that the `TomlConfig` object in `__init__` is indeed from a `pyproject.toml` file?

Reply:
Did the switch back